docs/_newsfragments/2100.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fix an issue when using orjson as json serializer with asyncio.
```

Comment:
This newsfragment could be improved: we shouldn't use the imperative mood here (as in docstrings), but tell a story what has been done. Also, we should capitalize JSON, and that `asyncio` sounds a bit vague, maybe we could be more explicit that it is `falcon.asgi.App` we're talking about.

Reply:
updated. hopefully it's better than before